modules/nw-kernel-bonding.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * networking/advanced_networking/network-bonding-considerations.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-kernel-bonding.adoc_{context}"]
+= Kernel bonding
+
+Kernel bonding is a built-in Linux kernel function where link aggregation can exist among many Ethernet interfaces to create a single logical physical interface. Kernel bonding is the default mode if no bond interfaces depend on OVS bonds. This bonding type does not give the same level of customization as supported OVS bonding.
+
+For `kernel-bonding` mode, the bond interfaces exist outside, which means they are not in the data path, of the bridge interface. Network traffic in this mode is not sent or received on the bond interface port but instead requires additional bridging capabilities for MAC address assignment at the kernel level. 
+
+If you enabled `kernel-bonding` mode on network controller interfaces (NICs) for your nodes, you must specify a Media Access Control (MAC) address failover. This configuration prevents node communication issues with the bond interfaces, such as `eno1f0` and `eno2f0`.
+
+Red Hat supports only the following value for the `fail_over_mac` parameter:
+
+* `0`: Specifies the `none` value and this is the default value that disables MAC address failover so that all compute nodes receive the same MAC address as the bond interface. With this setting, packets might be sent to inactive nodes.
+
+Red Hat does not support the following values for the `fail_over_mac` parameter:
+
+* `1`: Specifies the `active` value and sets the MAC address of the primary bond interface to always remain the same as active nodes. If during a failover, the MAC address of a node changes, the MAC address of the bond interface changes to match the new MAC address of the node.
+* `2`: Specifies the `follow` value so that during a failover, an active node gets the MAC address of the bond interface and a formerly active node receives the MAC address of the newly active node.
+
```

Comment:
We could link to https://github.com/torvalds/linux/blob/master/Documentation/networking/bonding.rst

Reply:
We cannot link to the KBS article that don't have approvals.